src/electron/biometric.main.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    KeyCredentialCreationOption,
+    KeyCredentialManager,
+    KeyCredentialStatus,
+} from '@nodert-win10-rs4/windows.security.credentials';
+import { ipcMain } from 'electron';
+import * as util from 'util';
+import { StorageService } from '../abstractions';
+import { ElectronConstants } from './electronConstants';
+
+const requestCreateAsync = util.promisify(
+    KeyCredentialManager.requestCreateAsync,
+);
+
+export class BiometricMain {
+    constructor(private storageService: StorageService) {}
+
+    async init() {
+        this.storageService.save(ElectronConstants.enableBiometric, await this.supportsBiometric());
```

Comment:
Why not just ipc like the request one? Not suggesting it, just wondering why?

Reply:
It seemed like a better solution to store it rather than constantly calling the method.